diff --git a/CMakeLists.txt b/CMakeLists.txt
index 52e9edb..c864b62 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -74,10 +74,6 @@ elseif (WIN32)
     add_definitions(-DLEVELDB_PLATFORM_WINDOWS "-DDLLX=__declspec(dllexport)")
 endif()
 
-add_library(leveldb SHARED ${SOURCES})
-
-find_package(ZLIB REQUIRED)
-if (ZLIB_FOUND)
-    include_directories( ${ZLIB_INCLUDE_DIRS} )
-    target_link_libraries( leveldb ${ZLIB_LIBRARIES} )
-endif(ZLIB_FOUND)
+add_library(leveldb STATIC ${SOURCES})
+include_directories( ${ZLIB_INCLUDE_DIRS} )
+target_link_libraries( leveldb ${ZLIB_LIBRARIES} )
